src/components/ListOfUsers.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./App.css";
+import users from "../users";
+import Button from "./Button";
+import UserDetail from "./UserDetail";
+
+function ListOfUsers() {
+  const allUsers = users.map(printUsers);
+  return (
+    <div>{allUsers}</div>
+  );
+}
+
+function printUsers(user) {
```

Comment:
Naming: In react, the users are not really printed... render would probably be more accurate